2-mandatory/2-financial-times.js

```diff
@@ -14,7 +22,18 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO: for each article string, split the string and count the number of words which should equal the number of array items. If the current number of array items is less than the value stored in the  shortestTitle variable, set current article title as the new value.
+  let shortestTitle = "";
+  for (const article of allArticleTitles) {
+    let wordsInTitleArr = article.split(" ");
+    let shortestTitleArr = shortestTitle.split(" ");
+    let wordCount = wordsInTitleArr.length;
+    let shortestTitleCount = shortestTitleArr.length;
+    if (wordCount < shortestTitleCount || shortestTitle === "") {
+      shortestTitle = article;
+    }
+  }
+  return shortestTitle;
 }
```

Comment:
amazing thinking finding the solution, it took a long time for me. 
one tip would be to do more frequent git push.